agent/php_user_instrument.c

```diff
@@ -88,10 +93,64 @@ int nr_zend_call_orig_execute_special(nruserfn_t* wraprec,
  * (or can't) match by name and have the zend_function available. (The main use
  * case for this is to allow instrumenting closures, but it's useful anywhere
  * we're dealing with a callable rather than a name.)
+ *
+ * There are two main structures containing `wraprecs`.
+ * `nr_php_op_array_set_wraprec` is a list of pointers to `wraprecs` that will
```

Comment:
Is the name `nr_php_op_array_set_wraprec` correct? Isn't it the name of the function that stores an index into `user_function_wrappers` vector defined here: [agent/php_newrelic.h:487](https://github.com/newrelic/newrelic-php-agent/blob/main/agent/php_newrelic.h#L487)? Aren't `user_function_wrappers` vector and `nr_wrapped_user_functions` list (defined in [agent/php_user_instrument.c:436](https://github.com/newrelic/newrelic-php-agent/blob/main/agent/php_user_instrument.c#L436)) the two structures that hold information about `wraprecs`?

Reply:
Good catch, thanks for reading!